setup.cfg

```diff
@@ -2,4 +2,4 @@
 universal=1
 
 [egg_info]
-egg_base = /tmp
+egg_base = tmp
```

Comment:
The reason this exists is so the `.egg-info` doesn't get copied into the root folder. Merge to TIP will use `./build/python` instead. Does that work?